### DISCUSSION TITLE
Start of the statistics module

### DISCUSSION THREAD

**Discussion by modir** [shopware/platform#3146] [RFC / Proposal]
**Effort**
low

### Description
It would be great if the foundation for a statistics module would be made.

### Benefits
It makes it then easier for others to extend the code and add their own statistics.

### Break strategy
If there is any break then only because some code from the dashboard has to move to this new module.

**Comment by lasomethingsomething** [comment]
Thank you for the idea, @modir. Due to the age of this thread and lack of activity in recent years, I will close.

**Reply by JoshuaBehrens** [reply]
Would it be worth mentioning, that there is the Analytics app and the way to get new graphs in there?

**Reply by mitelg** [reply]
@JoshuaBehrens absolutely. https://store.shopware.com/en/swag541977532977f/shopware-analytics.html
